### PR TITLE
Module clean: Removed unused local variables

### PR DESCRIPTION
**Pull request by andygrunwald**
This PR doesn't contain functionality changes.
It is only about code cleanup:

* Removed unused local variables

### CODE REVIEW

File: modules/install/class_import.php [hunk 1]
```diff
@@ -47,8 +47,7 @@ public function GetImportHeader($tmp_file_name)
     // See: http://lansuite.orgapage.de/index.php?mod=bugtracker&bugid=1059
     #$this->xml_content = str_replace("''", "'", $this->xml_content);
 
-        ## Get Header-Tag
-        $header = "";
+        // Get Header-Tag
         $this->xml_content_lansuite = $xml->getFirstTagContent("LANsurfer", $this->xml_content, 0);
```
Comment (M4LuZ) [inline modules/install/class_import.php]: I think we can remove the whole LANsurfer code. Project died in 2006 as far as I can see. Not sure anybody would need an import function still. And if they do: they can import with LanSuite 4.2 and then move forward from there

Comment (andygrunwald) [reply modules/install/class_import.php]: Will do. Let me create a ticket for this and handle this in a separate PR.

File: modules/signon/class_signon.php [hunk 2]
```diff
@@ -80,10 +80,9 @@ public function GetLansurfer($username, $password)
             fclose($handle);
 
             // Log out
-            $handle = fopen("http://lansurfer.com/user.phtml?action=logout?LS_Session=$LS_Session", 'r', false, stream_context_create($opts));
+            fopen("http://lansurfer.com/user.phtml?action=logout?LS_Session=$LS_Session", 'r', false, stream_context_create($opts));
```
Comment (M4LuZ) [inline modules/signon/class_signon.php]: See above. Would remove code related to lansufer completely.

File: modules/install/class_import.php (hunk 1 above)
Comment (andygrunwald) [reply modules/install/class_import.php]: @M4LuZ As a follow up: https://github.com/lansuite/lansuite/issues/222

File: modules/signon/class_signon.php (hunk 2 above)
Comment (andygrunwald) [reply modules/signon/class_signon.php]: See https://github.com/lansuite/lansuite/issues/222

**Comment by andygrunwald** [comment]
@M4LuZ Conflict resolved.

**Comment by andygrunwald** [comment]
Conflict resolved. When the tests are green, we are good to go.

**Pull request merged**